src/ozone/wayland/screen.cc

```diff
@@ -73,7 +73,6 @@ void WaylandScreen::OutputHandleMode(void* data,
                                      int32_t width,
                                      int32_t height,
                                      int32_t refresh) {
-  WaylandScreen* screen = static_cast<WaylandScreen*>(data);
```

Comment:
that's gonna crash. Check line 79

Reply:
There is another definition in the line 78.

Reply:
oh, indeed